ampd/src/queue/queued_broadcaster.rs

```diff
@@ -130,29 +125,26 @@ where
                 }
               },
               _ = interval.tick() => {
-                broadcast_all(&mut queue, &mut broadcaster).await?;
-                interval.reset();
-              },
-              _ = self.broadcast_rx.recv() => {
```

Comment:
when was this case being triggered?

Reply:
There was a functionality to force broadcast which was never used. Removed it for simplicity.